validator/validators/performance/performance_stats.go

```diff
@@ -0,0 +1,63 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package performance
+
+import (
+	"log"
+	"math"
+	"sort"
+)
+
+type Stats struct {
```

Comment:
The statistics metrics that customers will get when showing with the frontend page.[ Porting from intern projects](https://github.com/aws/amazon-cloudwatch-agent-test/blob/d5105cdc461c6fcb13049cf2d38c287674d94e21/test/performancetest/performance_query_utils.go#L49-L57)

Reply:
I think this should include more than just P99. Why not add P95, P90?
It is easy to add more stats in the future, or is changing it "hard"?

Reply:
Yes its easy to do so. That's the good thing about non-SQL Database. We don't need to define the structure for it beforehand but only some index. Therefore, easily adding with changing the calculations